<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>256</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>11</y>
     <width>120</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_X">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>X：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_X">
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="minimum">
       <double>-9.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_mm_2">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Y">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Y：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_Y">
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="minimum">
       <double>-9.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_o_2">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>131</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>36</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>36</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>距离：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_distance">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_mm">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>36</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>36</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>角度：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_angle">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_o">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>°</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>逆时针：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_AntiClockwise">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_mm_3">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>°</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>顺时针：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_Clockwise">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_o_3">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>°</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
